src/build.sh

```diff
@@ -154,6 +164,17 @@ function kernel_build()
   return "$ret"
 }
 
+# This function adds the build clean option
+# integrated to env in order to clean the kernel.
+function build_clean()
+{
+  local output_kbuild_flag="$1"
+  local flag
```

Comment:
You should receive flag as a parameter; search for this flag utilization in the code. After you understand that, you will notice how easy it is to test this function.